PreFreeSurfer/scripts/T2wToT1wDistortionCorrectAndReg.sh

```diff
@@ -64,9 +67,13 @@ opts_AddMandatory '--method' 'DistortionCorrection' 'method' "method used for re
 
         '${PHILIPS_METHOD_OPT}'
            use Philips specific Gradient Echo Field Maps for readout distortion correction
-
+        
         '${GENERAL_ELECTRIC_METHOD_OPT}'
-           use General Electric specific Gradient Echo Field Maps for readout distortion correction
+           use GE HealthCare Legacy specific Gradient Echo Field Maps for SDC (fieldmap and magnitude in a single NIfTI file via --fmapgeneralelectric argument).
+           This option is maintained for backward compatibility.
+
+        '${GE_HEALTHCARE_METHOD_OPT}'
+             use GE HealthCare specific Gradient Echo Field Maps for SDC (fieldmap in Hz and magnitude in two separate NIfTI files via --fmapphase and --fmapmag).
```

Comment:
Indentation here doesn't match the other help info.  Same comment applies to the help info parts in other scripts, but the expected indentation is also different than here...

Reply:
```suggestion
           use GE HealthCare specific Gradient Echo Field Maps for SDC (fieldmap in Hz and magnitude in two separate NIfTI files via --fmapphase and --fmapmag).
```